sim/druid/talents.go

```diff
@@ -106,6 +93,63 @@ func (druid *Druid) ApplyTalents() {
 	druid.applyPrimalMadness()
 }
 
+func (druid *Druid) applyBalanceOfPower() {
+	if druid.Talents.BalanceOfPower > 0 {
+		druid.AddStaticMod(core.SpellModConfig{
+			School:     core.SpellSchoolArcane | core.SpellSchoolNature,
+			ClassMask:  DruidArcaneSpells | DruidNatureSpells,
```

Comment:
DruidAllSpells with the above school masks will achieve the same. Those 2 masks can be completely removed

Reply:
removed

Reply:
You can have as many grouped masks as you want in reality. If they provide you with some benefit then yeah you can keep them. Its just to me it looked like they dont really do much as school masks are already a thing in spell mod/proc trigger filters but if you say its handy to have them just keep em yea